app/models/appointment_summary.rb

```diff
@@ -10,4 +10,5 @@ class AppointmentSummary < ActiveRecord::Base
   validates :income_in_retirement, inclusion: { in: %w(pension other) }
   validates :guider_name, presence: true
   validates :guider_organisation, inclusion: { in: %w(tpas cab) }
+  validates :reference_number, presence: true
```

Comment:
The database field is an integer so it would make sense to validate the numericality too.